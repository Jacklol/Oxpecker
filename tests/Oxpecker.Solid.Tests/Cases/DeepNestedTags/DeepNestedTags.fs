module Oxpecker.Solid.Tests.Cases.DeepNestedTags

open Oxpecker.Solid

[<SolidComponent>]
let DeepNestedTags () =
    body() {
        div(id="outer") {
            div(id="inner") {
                h1() { "Hello!" }
            }
        }
    }
