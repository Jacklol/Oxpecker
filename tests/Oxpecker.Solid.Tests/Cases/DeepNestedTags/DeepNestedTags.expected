
export function DeepNestedTags() {
  return <body>
    <div id="outer">
      <div id="inner">
        <h1>
            Hello!
        </h1>
      </div>
    </div>
  </body>;
}

